src/components/common/YourCompany/BusinessContactInfo.vue

```diff
@@ -6,7 +6,6 @@
     :hasBusinessContactInfoChange="hasBusinessContactInfoChange"
     :editLabel="editLabel"
     :editedLabel="editSavedLabel"
-    :disableActions="isCorrectionFiling"
```

Comment:
I think you also have to update the if block on lines 67+ to allow BEN correction filings to make the update.